ethstats/ethstats.go

```diff
@@ -181,11 +217,17 @@ func (s *Service) loop() {
 	txSub := txpool.SubscribeNewTxsEvent(txEventCh)
 	defer txSub.Unsubscribe()
 
+	istDelegateSignCh := make(chan istanbul.MessageEvent, istDelegateSignChanSize)
+	istDelegateSignSub := s.backend.SubscribeNewDelegateSignEvent(istDelegateSignCh)
+	defer istDelegateSignSub.Unsubscribe()
+
 	// Start a goroutine that exhausts the subsciptions to avoid events piling up
 	var (
 		quitCh = make(chan struct{})
 		headCh = make(chan *types.Block, 1)
 		txCh   = make(chan struct{}, 1)
+		signCh = make(chan StatsPayload, istDelegateSignChanSize)
```

Comment:
Should this be of type `chan *StatsPayload`?

I suppose it's not a huge deal, but would save a bit on memory usage by passing by reference vs. passing by value.